library/std/src/lib.rs

```diff
@@ -210,6 +210,8 @@
 #![allow(unused_lifetimes)]
 // Tell the compiler to link to either panic_abort or panic_unwind
 #![needs_panic_runtime]
+// Ensure that std can be linked against panic_abort despite compiled with `-C panic=unwind`
+#![cfg_attr(not(bootstrap), deny(ffi_unwind_calls))]
```

Comment:
Don't `core` and `alloc` need the same?

Reply:
I actually want to make it a default for compiling std and all its dependencies -- but not sure how.

Reply:
@nbdd0121 was this followed up on? In the future, please feel free to reach out to #t-infra on Zulip for questions like this, we can help with the instrumentation necessary. In this case you'd likely add to the deny'd lints in bootstrap, around [here](https://github.com/rust-lang/rust/blob/53792b9c5c3cded466a073e4b9a9e51d0007b9ab/src/bootstrap/builder.rs#L1885), with a gate on Mode::Std.

Reply:
I created a [thread](https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/Enable.20a.20lint.20when.20compiling.20std.20dependencies) in t-libs. The lint should be denied for std and its dependencies but not denied for panic runtimes, and it seems that this isn't quite achievement at the moment.